src/three-components/ModelScene.js

```diff
@@ -47,6 +37,7 @@ const FOV = 45;
 const DPR = window.devicePixelRatio;
 
 const $paused = Symbol('paused');
+const tempMat4 = new Matrix4();
```

Comment:
let's just call this lowercase `matrix4`

Reply:
:+1: